experimental/resource.go

```diff
@@ -0,0 +1,49 @@
+package experimental
+
+import (
+	"encoding/json"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// SendPlainText returns a plain text snippit
+func SendPlainText(sender backend.CallResourceResponseSender, text string) error {
+	return SendResourceResponse(
+		sender,
+		200,
+		map[string][]string{
+			"content-type": {"text/plain"},
+		},
+		[]byte(text),
+	)
+}
+
+// SendJSON returns a json object
```

Comment:
It's possible that the function returns something in the logical sense, but to say that "SendJSON" returns something at least doesn't correspond very well with its name :)
```suggestion
// SendJSON sends a JSON object.
```